package.json

```diff
@@ -1,21 +1,22 @@
 {
-  "name": "kompanion",
-  "version": "1.0.0",
-  "main": "index.js",
-  "license": "MIT",
-  "private": true,
-  "scripts": {
-    "build:client": "yarn workspace client build",
-    "build:website": "yarn workspace website build",
-    "dev:client": "yarn workspace client dev",
-    "dev:website": "yarn workspace website dev",
-    "start:client": "yarn workspace client start"
-  },
-  "workspaces": [
-    "client",
-    "website"
-  ],
-  "devDependencies": {
-    "@biomejs/biome": "^1.5.3"
-  }
+    "name": "kompanion",
+    "version": "1.0.0",
+    "main": "index.js",
+    "license": "MIT",
+    "private": true,
+    "scripts": {
+        "build:client": "yarn workspace client build",
+        "build:website": "yarn workspace website build",
+        "dev:client": "yarn workspace client dev",
+        "dev:website": "yarn workspace website dev",
+        "start:client": "yarn workspace client start",
+        "setup:all": "yarn install && yarn workspace client prisma migrate deploy && yarn workspace client prisma generate"
+    },
+    "workspaces": [
+        "client",
+        "website"
+    ],
+    "devDependencies": {
+        "@biomejs/biome": "^1.5.3"
+    }
```

Comment:
Looks like your editor is using 4 spaces for indent. Do you have the biome vscode extension installed?

Could you replace the default formatter with that?

Reply:
will do. we could create a vscode workspace for the project with the preferred tools/extensions